python_ta/typecheck/base.py

```diff
@@ -185,17 +230,17 @@ def unify(self, t1, t2):
         elif issubclass(t1, t2) or issubclass(t2, t1):
             pass
         elif t1 != t2:
-            raise Exception(str(t1) + ' ' + str(t2))
+            raise BadUnificationError(str(t1) + ' ' + str(t2))
 
-    def _unify_generic(self, t1: GenericMeta, t2: GenericMeta):
-        """Unify two generic types."""
+    def _unify_generic(self, t1, t2):
```

Comment:
It is regression to remove a type signature without replacing it with an updated one. Don't do this.